packages/components/src/internal/actions.ts

```diff
@@ -1595,19 +1300,6 @@ function getCopyValue(model: EditorModel, insertColumns: QueryColumn[]): string
     return copyValue;
 }
 
-export function unfocusCellSelection(modelId: string): void {
```

Comment:
I take it these were missed being removed in a previous refactor for EditableGrid?

Reply:
Yes, I believe so.